src/Tribe/Main.php

```diff
@@ -3828,17 +3836,25 @@ public function isVenue( $postId = null ) {
 		 * @return bool Is it an organizer?
 		 */
 		public function isOrganizer( $postId = null ) {
+			static $is_organizer = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 59f9db1112aa8645f8604cd118acdcaf</sub>

Reply:
Ignoring.